src/content/faq/General.md

```diff
@@ -8,20 +8,76 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Why should I use GraphQL?
 
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance and stability by [sending queries](/learn/queries/) for the exact data you need. 
```

Comment:
I am not sure what stability means here. Can we either explain it better or remove it?

Reply:
Removed!